source/Host/common/Socket.cpp

```diff
@@ -363,7 +363,7 @@ Error Socket::Close() {
   Log *log(lldb_private::GetLogIfAnyCategoriesSet(LIBLLDB_LOG_CONNECTION));
   if (log)
     log->Printf("%p Socket::Close (fd = %i)", static_cast<void *>(this),
-                m_socket);
+                static_cast<int>(m_socket));
```

Comment:
Socket is defined as:
```
typedef UINT_PTR        SOCKET;
```
The warning we therefore get is trying to convert `unsigned long long` to `int`